src/NServiceBus.RabbitMQ/RabbitMqDequeueStrategy.cs

```diff
@@ -156,7 +154,7 @@ void ConsumeMessages(object state)
 
                     var consumer = new QueueingBasicConsumer(channel);
 
-                    channel.BasicConsume(parameters.Queue, noAck,receiveOptions.ConsumerTag, consumer);
+                    channel.BasicConsume(parameters.Queue, false, receiveOptions.ConsumerTag, consumer);
```

Comment:
Since tx == off means "fire and forget" we need to ack right away "if txmode == none" ?